index.html

```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Williams David</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/style.css">
+
 </head>
+
 <body>
-    
+    <div class="image">
+        <nav>
+            <div class="navbar-center">
+                <ul>
+                    <li>
+                        <a href="/">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="/">About</a>
+                    </li>
+                    <li>
+                        <a href="/">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="mobile">
+                <ul>
+                    <li><a>Welcome</a></li>
+                    <li>
+                        <a class="menu"><i class="fa-solid fa-bars"></i></a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </div>
```

Comment:
- [ ] Please the container of the section is a `<header>` tag instead of `<div>` because it pecifies a header for a document or section.

Reply:
Ok I got it, ty